pom.xml

```diff
@@ -1,5 +1,17 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>com.github.jnr</groupId>
+            <artifactId>jnr-constants</artifactId>
+            <version>0.9.9</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.jnr</groupId>
+            <artifactId>jnr-posix</artifactId>
+            <version>3.0.41</version>
```

Comment:
I'm a bit confused; this is simply reverting to the approached used prior to 3c10cc0; wouldn't this argument had held prior to this 3c10cc0 as well? I'm admittedly rather unfamiliar with the Jenkins class loading scheme; under what circumstances would the version pushed to the builder differ from version required here?